### PR TITLE
Increase the fluent-bit priority class value to 150

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement

**What this PR does / why we need it**:
With this PR we increase the priority class value of fluent-bit to be greater than the control-plane one.
This is needed because when we have a specifically dedicated worker pool in which all the nodes have exceeded their pod number capacity the fluent-bit pod could not be scheduled. This will allow fluent-bit to preempt one of the control-plane components.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
Fluent-bit priority class value is increased from 50 to 150
```

### CODE REVIEW

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@ apiVersion: {{ include "priorityclassversion" . }}
 kind: PriorityClass
 metadata:
   name: fluent-bit
-value: 50
+value: 101
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml]: What do you think to set it to 120 or 150, just to have some free pod priorities for future use between fluent-bit and gardener-shoot-controlplane?

File: pkg/operation/seed/seed.go [hunk 2]
```diff
@@ -422,6 +422,17 @@ func RunReconcileSeedFlow(
...
+		//TODO: Remove after next release
+		flbpc := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit"}}
+		if err := k8sSeedClient.Client().Get(ctx, kutil.Key("fluent-bit"), flbpc); client.IgnoreNotFound(err) != nil {
+			return err
+		}
+		if (err == nil || client.IgnoreNotFound(err) == nil) && flbpc.Value < 101 {
+			if err := k8sSeedClient.Client().Delete(ctx, flbpc); client.IgnoreNotFound(err) != nil {
+				return err
+			}
+		}
```
Comment (vpnachev) [inline pkg/operation/seed/seed.go]: Just choose the right value
```suggestion
		if err := deletePriorityClassIfValueNotTheSame(ctx, k8sSeedClient.Client(), "fluent-bit", 120); err != nil {
			return err
		}
```

**Review by vpnachev — CHANGES_REQUESTED** [review]

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml (hunk 1 above)
Comment (vlvasilev) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml]: Good point. Thanks

File: pkg/operation/seed/seed.go (hunk 2 above)
Comment (vlvasilev) [reply pkg/operation/seed/seed.go]: thanks

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**